__tests__/app.test.js

```diff
@@ -48,4 +49,36 @@ describe('/api (getting all endpoints)', () => {
         });
     });
 });
+
+describe('/api/articles/:article_id', () => {
+    test('making sure that its responding with an article with the desired id', () => {
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then((article) => {
+            expect(article.body.article.article_id).toBe(2);
+            expect(Object.keys(article.body.article))
```

Comment:
Ideally we would want to know what values are there too, we can either check for specific values or for the data type of the value (I.E. expect any number type of syntax). You could use objectContaining as well 🙂